hedera-node/src/main/java/com/hedera/services/exceptions/NoValidSignedStateException.java

```diff
@@ -0,0 +1,30 @@
+package com.hedera.services.exceptions;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+/**
+ * Captures a failure when an invalid signed state is provided to update state children
+ */
+public class NoValidSignedStateException extends Exception {
+	public NoValidSignedStateException(String message) {
```

Comment:
Will the `message` add anything here?

Reply:
Ah it is self explanatory 😞 . Removing it